frontend/src/app/App.scss

```diff
@@ -64,6 +64,14 @@ body,
   }
 }
 
+// giving a fixed width and height of the icon for app launcher
+.application-launcher-menu-item-icon {
+  .pf-v5-c-menu__item-icon {
+    width: 1.5rem;
+    height: 1.5rem;
+  }
+}
+
```

Comment:
Replacing the deprecated application launcher with a dropdown, noticed that the icons for dropdown items were not rendering. This is because the deprecated application launcher item icons had a fixed width and height specified,  That is the reason added css style with same dimensions

Reply:
The icons render fine in the PF example: https://www.patternfly.org/components/menus/application-launcher#application-launcher-menu

Can you post screenshots please. 

I'd prefer to not have custom CSS.

Reply:
@christianvogt , yes I agree that icon renders fine thats because there have a fixed size but the svg image used here rely on the container size, the deprecated application launcher item icon had the container size specified
<img width="750" alt="Screenshot 2024-06-24 at 6 45 58 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/99238909/9bb65c9e-ece6-44a6-b5d6-6502863b791e">

<img width="327" alt="Screenshot 2024-06-27 at 4 08 21 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/99238909/7f7ec0b3-17bf-44c8-86d3-d259175e31d6">

Reply:
After investigating with the PF team it turns out there's no good solution here other than to specify a size.

Please move this rule to `AppLauncher.scss` and rename to `.odh-app-launcher__dropdown-item` to fit naming conventions.